prow/config.yaml

```diff
@@ -1622,6 +1622,72 @@ presubmits:
       - name: docker-graph
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
+  - name: pull-kubernetes-dind-e2e
+    agent: kubernetes
+    context: pull-kubernetes-dind-e2e
+    always_run: false
+    rerun_command: "/test pull-kubernetes-dind-e2e"
+    trigger: "(?m)^/test pull-kubernetes-dind-e2e,?(\\s+|$)"
+    skip_branches:
+    - release-1.6
+    - release-1.7
+    - release-1.8
+    - release-1.9
+    labels:
+      preset-service-account: true
+      preset-bazel-scratch-dir: true
+      preset-bazel-remote-cache-enabled: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20180215-b2a89850e
+        args:
+        - "--clean"
+        - "--git-cache=/root/.cache/git"
+        - "--job=$(JOB_NAME)"
+        - "--repo=k8s.io/kubernetes=master"
+        - "--repo=k8s.io/test-infra=master"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=execute"
+        - "--"
+        - "kubetest"
+        - "--build=dind"
+        - "--deployment=dind"
+        - "--up"
+        - "--test"
+        - "--down"
+        # Bazel needs privileged mode in order to sandbox builds.
+        securityContext:
+          privileged: true
+          capabilities:
+            add: ["SYS_ADMIN"]
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: modules
+        - mountPath: /sys/fs/cgroup
+          name: cgroup
+        # TODO(bentheelder): eliminate the "git caching" junk
+        - name: cache-ssd
+          mountPath: /root/.cache
+        ports:
+        - containerPort: 9999
```

Comment:
:(((( pls no

Reply:
Is there an alternative way?

Reply:
yeah, just disable git caching, don't mount host volumes, use an emptyDir for the dind storage.